packages/wdio-cli/src/launcher.js

```diff
@@ -102,6 +102,15 @@ class Launcher {
             })
         }
 
+        /**
+         * fail if no caps were found
+         */
+        if (!caps.length) {
```

Comment:
We should add a UT for this

Reply:
Can you explain what you mean by a UT? ;)

Reply:
Unit Test.

Reply:
Unit Test - If this is a new check, we should test it.